### ISSUE TITLE
Angular project compilation fails using latest version of ngx-flicking

### ISSUE THREAD

**Issue by alechemy** [naver/egjs-flicking#574]
## Description
Building an Angular application will fail after installing the latest version of `ngx-flicking`.

The following error is thrown:

```
Error: Failed to compile entry-point @egjs/ngx-flicking (`es2015` as esm2015) due to compilation errors:
node_modules/@egjs/ngx-flicking/fesm2015/egjs-ngx-flicking.js:282:18 - error NG1010: @HostBinding's argument must be a string
  Value could not be determined statically.

282     isVertical: [{ type: HostBinding, args: [`class.${CLASS.VERTICAL}`,] }],
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@egjs/ngx-flicking/fesm2015/egjs-ngx-flicking.js:282:55
    282     isVertical: [{ type: HostBinding, args: [`class.${CLASS.VERTICAL}`,] }],
                                                              ~~~~~~~~~~~~~~
    Unable to evaluate this expression statically.
  node_modules/@egjs/flicking/declaration/const/external.d.ts:36:22
     36 export declare const CLASS: {
                             ~~~~~~~~
     37     VERTICAL: string;
        ~~~~~~~~~~~~~~~~~~~~~
    ... 
     39     DEFAULT_VIRTUAL: string;
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     40 };
        ~
    A value for 'CLASS' cannot be determined statically, as it is an external declaration.
```

It looks like that `HostBinding` was recently introduced, so I suspect that introduced this issue.
https://github.com/naver/egjs-flicking/commit/b21f5939fd998bb8146086936df80e9a9d20d01f#diff-a8a26d048f66449e6f4ab8e283e6478ba03a9f6b13bc8b9b1f023f8d500ecdb3R102

## Steps to check or reproduce
Here's a sample stackblitz: https://stackblitz.com/edit/angular-12-template-tggqsf?file=src/app/app.module.ts

**Comment by WoodNeck** [comment]
Hello @alechemy! Thanks for the reporting!
I'll fix this asap!

**Comment by alechemy** [comment]
Thanks. I saw that you opened a PR and it has been approved. Anything else holding it back? This bug is blocking my team from using the library.

**Comment by WoodNeck** [comment]
@alechemy 
Sorry for the wait! I've just released v4.4.1 that fixes this issue, please try that!

**Comment by alechemy** [comment]
Works fine now. Thanks for the quick fix.
